app/controllers/users/invitations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Users::InvitationsController < Devise::InvitationsController
+  before_action :add_organisation_to_params, only: :create
+
+private
+
+  def add_organisation_to_params
+    params[:user][:organisation_id] = current_user.organisation_id
+  end
+
+  # Overrides https://github.com/scambra/devise_invitable/blob/master/app/controllers/devise/invitations_controller.rb#L105
```

Comment:
I wouldn't include this comment - I think the pattern of overriding Devise controllers is common enough that a reader will know what's going on.

Also, unlike before, I'm happy with this override - it's more clear that the intent of this class is to `add_organisation` at the point of inviting.

Reply:
No worries, I'm happy if you want to include it.